pipelines/datasets/br_fnde_fundeb/constants.py

```
@@ -0,0 +1,955 @@
+    """Constantes do conjunto br_fnde_fundeb.
+
+    Nome da classe em minúsculo segue a convenção do repo para enums de
+    constantes de dataset.
+    """
+
+    DATASET_ID = "br_fnde_fundeb"
+
+    TABLE_STATE = "indicador_estadual"
+    TABLE_MUNICIPALITY = "indicador_municipal"
+    TABLE_DICTIONARY = "dicionario"
+    ALL_TABLES = ["indicador_estadual", "indicador_municipal", "dicionario"]
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)